README.md

```diff
@@ -19,6 +19,25 @@ Because ML practitioners use so many different types of tools, it is a key goal
 
 Ultimately, we want to have a set of simple manifests that give you an easy to use ML stack _anywhere_ Kubernetes is already running and can self configure based on the cluster it deploys into.
 
+
+## Who should consider using Kubeflow?
```

Comment:
Shouldn't this be more generic that this? This sounds like its only restricted to these 3 cases but its not, correct?

Reply:
imo, the contributing section covers that nicely. but i agree it's always good to be clear about the intentions. i think the suggestion by @aronchick below is a nice nod in that direction.

Reply:
I wanted the list to reflect current capabilities where Kubeflow is a clear value add over competing solutions. Per David's comment I added a note below that this is based on current capabilities which we are working on expanding.